src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.ts

```diff
@@ -15,6 +15,7 @@ import type {
   SectionDescriptor,
 } from '../../../../inboxsdk';
 import * as s from './gmail-collapsible-section-view.module.css';
+import { sanitize } from 'dompurify';
```

Comment:
unused
```suggestion
```

Reply:
Dropped in [e4f07ed](https://github.com/InboxSDK/InboxSDK/pull/1133/commits/e4f07edb0b9f9a889953c2b48d2e03dc81fab6aa).

The reason it wasn't being used: `autoHTML` being used for labels and icons seems to break with `dompurify`'s `sanitize` function. I'll punt on sanitization for now. We may end up running into the same issue with #1127 though.